Automattic/jetpack
# Simple Payments Widget: Add Plan Check

Depends on:

- Simple Payments: adds plan support check for Jetpack and WordPress.com (#9650 and D14005-code)

#### Changes proposed in this Pull Request:

* Adds a plan support check to the SP Widget.

#### Testing instructions:

* Apply this branch on a Jetpack site with a Premium or Business Subscriptions.
* Using the Customizer, add a Simple Payment button as a widget to the site's sidebar.
* Navigate to site's frontend.

The Simple Payment button should display as a widget on the selected sidebar and location.

* Downgrade the site to the Free plan without deleting the SP Widget from the sidebar.
* Navigate to the Jetpack dashboard to refresh the site's settings.
* Navigate to site's frontend.

The site's sidebar should no longer show the Simple Payment.
